corehq/apps/app_manager/suite_xml/generator.py

```diff
@@ -148,6 +148,10 @@ def media_resources(self):
             install_path = '../../{}'.format(path)
             local_path = './{}/{}'.format(path, name)
 
+            load_lazily = False
+            lazy_load_preference = self.app.profile.get('properties', {}).get('lazy-load-multimedia-files')
+            if lazy_load_preference and lazy_load_preference == 'true':
+                load_lazily = True
```

Comment:
You could replace this with
`load_lazily = self.app.profile.get('properties', {}).get('lazy-load-multimedia-files') == 'true'`
or
```
load_lazily = (
    self.app.profile.get('properties', {}).
    get('lazy-load-multimedia-files') == 'true'
)
```
or if you like the break down
```
lazy_load_preference = self.app.profile.get('properties', {}).get('lazy-load-multimedia-files')
load_lazily = lazy_load_preference == 'true'
```

Reply:
The second snippet looks real nice, implemented that. 
Thanks for the suggestion. 😄